pages/common/fly.md

```diff
@@ -0,0 +1,35 @@
+# fly
+
+> Cli tool for concourse-ci.
+
+- Authenticate with and save concourse target:
+
+`fly --target {{example}} login --team-name {{my-team}} -c {{https://ci.example.com}}`
```

Comment:
We should probably rename example to sth like target_name so that in the later examples, its more apparent as to what is this.

Reply:
agreed, also applied this to teams and and pipelines (team_name, piipeline_name)